packages/react-components/src/components/ChatMessage/ChatMessageComponentAsEditBox.tsx

```diff
@@ -134,6 +134,11 @@ export const ChatMessageComponentAsEditBox = (props: ChatMessageComponentAsEditB
           placeholderText={strings.editBoxPlaceholderText}
           textValue={textValue}
           onChange={setText}
+          onKeyDown={(ev) => {
```

Comment:
Does this still enable any mention suggestion to be selected via up / down arrows?

Reply:
It shouldn't affect any existed function inside the Edit box, but just the FluentUI Chat Message behavior. 

`stopPropagation is a method that prevents the event from being propagated or "bubbling up" through the DOM.`

Reply:
And it was good when you tested it?

Reply:
Tested, and looks good